app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.worldsnas.starplayer.view.musics_list
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+import com.worldsnas.starplayer.R
+import com.worldsnas.starplayer.databinding.FragmentMusicsListBinding
+import com.worldsnas.starplayer.model.Music
+
+class MusicsListFragment : Fragment() {
+
+    private var _binding: FragmentMusicsListBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
fek mikonam niazi behesh nist

Reply:
Be chi? Oon dota variable binding?

Reply:
roye khat comment gozashtam
yani khate 18